src/const.js

```diff
@@ -12,5 +12,6 @@ export default {
     STAR: "star",
     TRIANGLE: "triangle",
     WYE: "wye",
+    RECTANGLE: "rectangle",
```

Comment:
You are introducing a new type of symbol that was precedently not supported by `react-d3-graph`. This should maybe not be in this Pull Request but in another like *Introducing Rectangle symbol* since it adds a new feature unsupported before. What do you think? 

I think that it also asks the question of whether the square symbol is still needed since a square is only a rectangle with four equal straight sides.

Reply:
> You are introducing a new type of symbol that was precedently not supported by react-d3-graph. This should maybe not be in this Pull Request but in another like Introducing Rectangle symbol since it adds a new feature unsupported before. What do you think?


That's a good point, but it's ok to ship the new `RECTANGLE` type in this PR, no need to separate it since our documentation will make that separation clear once we release a new version.

> I think that it also asks the question of whether the square symbol is still needed since a square is only a rectangle with four equal straight sides.

Makes total sense! Hey @lironhl do you mind going ahead and add a `@deprecated` [on the square here](https://github.com/danielcaldas/react-d3-graph/blob/adf5a1d1de9c0529bf024ffeae85b1ef1ddb06f7/src/components/graph/graph.config.js#L173:L173)? You can write a note saying that we can deprecate the type `"square"` in a next major version release.

Reply:
Hmmm, this is a difficult decision, I introduced the `rectangle` symbolType in order to be more explicit about the desired node shape, and to remind people to pass `{ width, height }` object to the `size` property of the `Node`, but I do not intend to support `rectangle` **SVG** shape (I looked into it briefly and couldn't find easy solution).

I think i'll go in @LonelyPrincess way, when she implemented `viewGenerator` property, in case the user passes the `{ width, height }` object to the `Node`s `size` property I'll consider that node as a **rectangle**, because an `{ width, height }` object do not mean anything when considering any other shape.

Allowing `rectangle` as a valid value in the `symbolType` property may create confusion because users will think it's also supported as an **SVG** shape

Reply:
Yes but I am afraid that looking at the code we will have the impression that this symbol can be used when it is just there as a *supported type for other purposes*. I think that either way what you are choosing (supporting it or not), this should be clearer with what can and can not be done with this new type (maybe a clearer name like *RECTANGLE_INTERNAL* to show that it can not be used by the users)?

Reply:
I see. I think I understand the dilemma now. @antoninklopp raised a good point though, how about making it so that the new type is used internally only. We could even rename it to `VIEW_GENERATOR_CROP` or something similar, in ways that it wouldn't be confused with a geometrical symbol type.